compose/galaxy-htcondor/Dockerfile

```diff
@@ -3,4 +3,8 @@ FROM quay.io/bgruening/galaxy-htcondor-base:18.09
 ADD condor_config.local /etc/condor/condor_config.local
 ADD supervisord.conf /etc/supervisord.conf
 
+RUN touch /var/log/condor/StartLog /var/log/condor/StarterLog /var/log/condor/CollectorLog /var/log/condor/NegotiatorLog && \
```

Comment:
Should we move this into the htcondor-base image?

Reply:
Done!